change_config_script.py

```diff
@@ -0,0 +1,22 @@
+import configparser
+import codecs
+import os
+from sioux import download
+
+# Get absolute path to config file in root directory
+config = configparser.ConfigParser()
+file_path = os.path.expanduser("~{0}.sioux{0}".format(os.path.sep)) + "config.cfg"
```

Comment:
You should use the constants from download.py for constructing this. Also use os.path.join instead of a `+` here.